Rubberduck.Parsing/Symbols/IdentifierReferenceResolver.cs

```diff
@@ -254,14 +247,17 @@ public void Resolve(VBAParser.RedimStmtContext context)
                 // It's not actually a function call but a special statement.
                 ResolveDefault(indexExpr.lExpression());
                 var argumentList = indexExpr.argumentList();
-                if (argumentList.argument() != null)
+
+                if (argumentList.argument() == null)
                 {
-                    foreach (var positionalArgument in argumentList.argument())
+                    continue;
```

Comment:
Do we really want to invert such conditions that lead to a `continue` or `break` statement?

Reply:
#itdepends, I guess. in this case the benefit seems marginal